ny-portal/src/app/(frontend)/komponenter/[slug]/components/ComponentHeaderLink.tsx

```diff
@@ -22,7 +22,13 @@ export const ComponentHeaderLink = ({
     return (
         <Tooltip>
             <TooltipTrigger>
-                <Button variant="secondary" as="a" href={href} icon={icon} />
+                <Button
+                    variant="secondary"
+                    as="a"
+                    target="_blank"
```

Comment:
Lenker som åpner i ny fane bør også ha noen ekstra egenskaper av sikkerhetshensyn:

```suggestion
                    target="_blank"
                    rel="noopener noreferrer"
```

Reply:
Jaaaa, for sikkerhets skyld. Opsi